.travis.yml

```diff
@@ -10,10 +10,12 @@ python:
   - "pypy"
 sudo: required
 group: deprecated-2017Q2
-services:
-  - docker
 before_install:
-  - docker run -d -p 10101:10101 pilosa/pilosa:v0.6.0
+  - sudo apt-get update
+  - sudo apt-get install python-pip
+  - sudo pip install awscli
```

Comment:
We don't need to install `awscli` if we just use S3 over HTTP: `wget https://s3.amazonaws.com/build.pilosa.com/pilosa-master-linux-amd64.tar.gz`

Reply:
Excellent! Updating.